### PR TITLE
chore: migrate to prettier 2.8

### PR DESCRIPTION
**Pull request by Hotell**
## Notes 💡

**Which migration causes most regressions ?:**

🔥 2.3

**Cant we enable options to mitigate so many diffs?:**

These formatting regression cannot be "undone" - there is no prettier config for those diffs to mitigate so many code changes.

All available options -> https://prettier.io/docs/en/options.html

**Can't we apply formatting gradually?:**

No -- why? --> We run format checks on whole codebase after every merge to master. While this will change in future, that's the current state.

## New Behavior

> 💡 For more granular diff check granular commits https://github.com/microsoft/fluentui/pull/26959/commits

### 2.3
- https://prettier.io/blog/2021/05/09/2.3.0.html
- most format changes between  2.2 -> 2.3 🚨
- diffs caused mainly by:
  - https://prettier.io/blog/2021/05/09/2.3.0.html#prevent-wrapping-object-properties-with-short-keys-10335httpsgithubcomprettierprettierpull10335-by-thorn0httpsgithubcomthorn0
  - https://prettier.io/blog/2021/05/09/2.3.0.html#format-assignments-more-consistently-10222httpsgithubcomprettierprettierpull10222-10643httpsgithubcomprettierprettierpull10643-10672httpsgithubcomprettierprettierpull10672-by-thorn0httpsgithubcomthorn0-10158httpsgithubcomprettierprettierpull10158-by-sosukesuzukihttpsgithubcomsosukesuzuki 

### 2.4
- https://prettier.io/blog/2021/09/09/2.4.0.html
- few format changes between 2.3 -> 2.4 🤷

### 2.5
- https://prettier.io/blog/2021/11/25/2.5.0.html
- few format changes between 2.4 -> 2.5 🤷

### 2.6
- https://prettier.io/blog/2022/03/16/2.6.0.html
- no format changes between 2.5 -> 2.6 ✅

### 2.7
- https://prettier.io/blog/2022/06/14/2.7.0.html
- no format changes between 2.6 -> 2.7 ✅

### 2.8
- https://prettier.io/blog/2022/11/23/2.8.0.html
- few format changes between 2.7 -> 2.8 🤷
- diffs caused mostly by:
  - https://github.com/prettier/prettier/blob/main/CHANGELOG.md#add-parentheses-for-tstypequery-to-improve-readability-14042-by-onishi-kohei

## Related Issue(s)

- Fixes partially https://github.com/microsoft/fluentui/issues/25488

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                          |        Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------- | --------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                 |        `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr> |        `554 B`<br />`348 B` |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr>   |  `3.194 kB`<br />`1.314 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                    | `23.113 kB`<br />`7.069 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                      | `24.002 kB`<br />`7.374 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                    | `25.438 kB`<br />`7.254 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                              |   `17.11 kB`<br />`6.18 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                    | `11.259 kB`<br />`4.473 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                    |  `9.822 kB`<br />`4.066 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                            | `11.214 kB`<br />`4.271 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                 | `22.985 kB`<br />`8.528 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                              |  `20.62 kB`<br />`6.682 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                     | `12.261 kB`<br />`4.819 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                   |  `15.571 kB`<br />`5.23 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>     |          `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>               | `30.908 kB`<br />`6.714 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>             | `17.776 kB`<br />`5.141 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                    |        `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/30ad74d01e550256a7f62dd8760d393d54473959'>30ad74d01e550256a7f62dd8760d393d54473959</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/BaseButton-mount_base.html">1202</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/BaseButton-mount.html">1217</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Breadcrumb-mount_base.html">2795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Breadcrumb-mount.html">2787</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Checkbox-mount_base.html">2665</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Checkbox-mount.html">2641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/CheckboxBase-mount_base.html">2378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/CheckboxBase-mount.html">2371</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ChoiceGroup-mount_base.html">4331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ChoiceGroup-mount.html">4297</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ComboBox-mount_base.html">1187</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ComboBox-mount.html">1189</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/CommandBar-mount_base.html">9291</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/CommandBar-mount.html">9299</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ContextualMenu-mount_base.html">12442</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ContextualMenu-mount.html">12468</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DefaultButton-mount_base.html">1384</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DefaultButton-mount.html">1369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DetailsRow-mount_base.html">3219</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DetailsRow-mount.html">3216</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DetailsRowFast-mount_base.html">3201</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DetailsRowFast-mount.html">3268</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DetailsRowNoStyles-mount_base.html">3093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DetailsRowNoStyles-mount.html">3170</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Dialog-mount_base.html">2926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Dialog-mount.html">2950</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DocumentCardTitle-mount_base.html">576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/DocumentCardTitle-mount.html">583</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Dropdown-mount_base.html">3126</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Dropdown-mount.html">3156</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/FocusTrapZone-mount_base.html">1919</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/FocusTrapZone-mount.html">1977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/FocusZone-mount_base.html">1925</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/FocusZone-mount.html">1923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedList-mount_base.html">47317</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedList-mount.html">52583</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedList-virtual-rerender_base.html">22285</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedList-virtual-rerender.html">22343</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">81525</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedList-virtual-rerender-with-unmount.html">81128</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedListV2-mount_base.html">558</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedListV2-mount.html">570</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedListV2-virtual-rerender_base.html">541</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedListV2-virtual-rerender.html">546</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">558</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/IconButton-mount_base.html">1786</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/IconButton-mount.html">1811</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Label-mount_base.html">755</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Label-mount.html">756</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Layer-mount_base.html">4160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Layer-mount.html">4181</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Link-mount_base.html">859</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Link-mount.html">845</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/MenuButton-mount_base.html">1596</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/MenuButton-mount.html">1628</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/MessageBar-mount_base.html">2376</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/MessageBar-mount.html">2332</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Nav-mount_base.html">3081</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Nav-mount.html">3087</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/OverflowSet-mount_base.html">1400</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/OverflowSet-mount.html">1421</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Panel-mount_base.html">2492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Panel-mount.html">2500</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Persona-mount_base.html">1300</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Persona-mount.html">1281</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Pivot-mount_base.html">1532</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Pivot-mount.html">1532</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/PrimaryButton-mount_base.html">1491</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/PrimaryButton-mount.html">1499</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Rating-mount_base.html">6890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Rating-mount.html">6971</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SearchBox-mount_base.html">1502</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SearchBox-mount.html">1496</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Shimmer-mount_base.html">2898</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Shimmer-mount.html">2893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Slider-mount_base.html">2118</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Slider-mount.html">2091</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SpinButton-mount_base.html">4282</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SpinButton-mount.html">4238</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Spinner-mount_base.html">829</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Spinner-mount.html">830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SplitButton-mount_base.html">2830</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SplitButton-mount.html">2846</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Stack-mount_base.html">871</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Stack-mount.html">867</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/StackWithIntrinsicChildren-mount_base.html">2252</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/StackWithIntrinsicChildren-mount.html">2228</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/StackWithTextChildren-mount_base.html">4928</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/StackWithTextChildren-mount.html">4905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SwatchColorPicker-mount_base.html">9473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/SwatchColorPicker-mount.html">9396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/TagPicker-mount_base.html">2298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/TagPicker-mount.html">2363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/TeachingBubble-mount_base.html">75492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/TeachingBubble-mount.html">75889</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Text-mount_base.html">823</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Text-mount.html">832</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/TextField-mount_base.html">1584</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/TextField-mount.html">1542</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ThemeProvider-mount_base.html">1435</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ThemeProvider-mount.html">1441</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ThemeProvider-virtual-rerender_base.html">1147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ThemeProvider-virtual-rerender.html">1139</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1988</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2000</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Toggle-mount_base.html">1137</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/Toggle-mount.html">1128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/buttonNative-mount_base.html">536</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26959/perf-test/buttonNative-mount.html">545</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

**Review by ling1726 — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

**Review by behowell — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]

File: apps/perf-test-react-components/src/scenarios/Persona.tsx [hunk 1]
```diff
@@ -10,8 +10,7 @@ const Scenario = () => (
     presence={{ status: 'available' }}
```
[the remark this answers is not in the document]
Comment (Hotell) [reply apps/perf-test-react-components/src/scenarios/Persona.tsx:10]: yeah not a very stable test results indeed.

**Pull request merged**